code/CMakeLists.txt

```diff
@@ -41,7 +41,12 @@ target_link_libraries(code PUBLIC md5)
 
 target_link_libraries(code PUBLIC discord-rpc)
 
-target_link_libraries(code PUBLIC libRocket)
+find_package(libRocket QUIET)
```

Comment:
To keep the "implementation" details concentrated to the site where the libRocket target is defined we usually define such libraries as interface targets with the same name as the one built from source. Then you could keep the same code here and still have it use the system version if it was found.